crates/ruff/src/rules/pycodestyle/rules/bare_except.rs

```diff
@@ -7,6 +7,32 @@ use crate::source_code::Locator;
 use crate::violation::Violation;
 
 define_violation!(
+    ///## What it does
```

Comment:
Doc comments should be indented with a space ... that is they should start with `/// `.

Reply:
fixed.